components/Cards/index.js

```diff
@@ -5,15 +5,68 @@
 // You will be creating a component for each 'article' in the list.
 // This won't be as easy as just iterating over an array though.
 // Create a function that will programmatically create the following DOM component:
-//
-// <div class="card">
+// //
+// {/* <div class="card">
 //   <div class="headline">{Headline of article}</div>
 //   <div class="author">
 //     <div class="img-container">
 //       <img src={url of authors image} />
 //     </div>
 //     <span>By {authors name}</span>
 //   </div>
-// </div>
+// </div> */}
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+.then(response => {
+
+const articles = Object.keys(response.data.articles);
+
+for (let i = 0; i < articles.length; i++) {
+        response.data.articles[articles[i]].forEach(element => {
+         new Card(element);  
+        });
+}
+
+
+console.log(articles);
+console.log(response.data);
+})
```

Comment:
It is always a good practice to chain a .catch function to handle errors when using promises